hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithFSO.java

```diff
@@ -218,37 +223,47 @@ public class TestNSSummaryEndpointWithFSO {
           + CONTAINER_TWO_REPLICA_COUNT * BLOCK_TWO_LENGTH
           + CONTAINER_THREE_REPLICA_COUNT * BLOCK_THREE_LENGTH;
 
-  private static final long MULTI_BLOCK_KEY_SIZE_WITH_REPLICA
-          = LOCATION_INFO_GROUP_ONE_SIZE;
 
   private static final long LOCATION_INFO_GROUP_TWO_SIZE
```

Comment:
If this is no longer being referenced, Can we remove it? Same for `LOCATION_INFO_GROUP_ONE_SIZE`

Reply:
yes, no longer referenced, could be removed!